changelogs/fragments/1673-return-job-type.yml

```diff
@@ -0,0 +1,10 @@
+minor_changes:
+  - zos_job_query - Added address space used by jobs in return JSON
```

Comment:
Would it be better to say ? 

```suggestion
  - zos_job_query - Added address space type used by jobs in return JSON
```

At first I thought it would return literal space, so expected a number.

Reply:
Agree, just changed it for every module in the fragment.